### PR TITLE
Add language-specific build rules.

### PR DESCRIPTION
**Pull request by DrMarcII**

### CODE REVIEW

File: web/dart.bzl [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  argument isn't listed below, then the argument values have the same
+  exact meaning as in dart_vm_test.
+
+  Args:
+    name: The base name of the test.
+    browsers: A sequence of lavels specifying the browsers to use.
+    args: Args for web_test targets generated by this extension.
+    browser_overrides: Dictionary mapping from browsers to browser-specific
+      attributes, such as shard_count, flakiness, timeout, etc. For example:
+      {'chrome-linux': {'shard_count': 3, 'flaky': 1}
```
Comment (juangj) [inline web/dart.bzl]: Use example browser names from this project, e.g., '//browsers:chromium-native'.

File: web/go.bzl [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+load("//web:web.bzl", "web_test_suite")
+
+
+def go_web_test_suite(name,
+                      browsers,
+                      args=None,
+                      browser_overrides=None,
+                      config=None,
+                      flaky=None,
+                      go_test_tags=("manual", "notap"),
```
Comment (juangj) [inline web/go.bzl]: Omit 'notap'.

File: web/web.bzl [hunk 3]
```diff
@@ -52,6 +53,8 @@ def web_test_suite(name,
       attributes, such as shard_count, flakiness, timeout, etc. For example:
       {'\\browsers:chrome-native': {'shard_count': 3, 'flaky': 1}
        '\\browsers:firefox-native': {'shard_count': 1, 'timeout': 100}}.
+    test_suite_tags: List of strings; optoinal default = ["manual"]; tags
```
Comment (juangj) [inline web/web.bzl]: s/optoinal/optional/

File: web/dart.bzl (hunk 1 above)
Comment (DrMarcII) [inline web/dart.bzl]: Done.

File: web/go.bzl (hunk 2 above)
Comment (DrMarcII) [inline web/go.bzl]: Done.

File: web/web.bzl (hunk 3 above)
Comment (DrMarcII) [inline web/web.bzl]: Done.

**Comment by DrMarcII** [comment]
I have updated the PR to create a new rule that contains most of the logic and make the specific language rules simpler.

**Pull request merged**